### PR TITLE
FIX Facture in supplier card

### PR DESCRIPTION
**Pull request by ptibogxiv**

### CODE REVIEW

File: htdocs/societe/class/societe.class.php [hunk 1]
```diff
@@ -3914,14 +3914,14 @@ public function getOutstandingBills($mode = 'customer')
 			while($obj=$this->db->fetch_object($resql)) {
                 $tmpobject->id=$obj->rowid;
 
-				if ($obj->fk_statut != Facture::STATUS_DRAFT                                           // Not a draft
+				if ($mode != 'supplier' && $obj->fk_statut != Facture::STATUS_DRAFT                                           // Not a draft
```
Comment (eldy) [inline htdocs/societe/class/societe.class.php]: Why adding test on mode != supplier ?
If we call method getOutstandingBills for a supplier invoice, we should get outstanding amount we have for this supplier.

Comment (ptibogxiv) [reply htdocs/societe/class/societe.class.php]: i try to fix this error on /fourn/card.php?socid=x

> Fatal error: Uncaught Error: Class 'Facture' not found in /volume2/web/test/dolibarr/htdocs/societe/class/societe.class.php:3917 Stack trace: #0 /volume2/web/test/dolibarr/htdocs/fourn/card.php(404): Societe->getOutstandingBills('supplier') #1 {main} thrown in /volume2/web/test/dolibarr/htdocs/societe/class/societe.class.php on line 3917

**Comment by eldy** [comment]
I see. Does this commit 23860b15ee9b09ee146696eec9d235c875a5097a
fix your trouble ? If yes, you can close the PR

**Comment by ptibogxiv** [comment]
@eldy, yes it's fixed with your PR. Thanks !

**Pull request closed without merging**